lib/middleware/use-async/use-async.js

```diff
@@ -1,23 +1,17 @@
-
 function useAsync (middleware) {
   return function wrappedRoute (req, res, next) {
-    middleware(req, res)
-      .then(() => {
-        // Express does not set headersSent immediately
-        process.nextTick(() => {
-          if (!res.headersSent) {
-            next()
-          }
-        })
-      })
-      .catch(err => {
-        process.nextTick(() => {
-          if (!res.headersSent) {
-            next(err)
-          }
-        })
+    const callNext = (err) => {
+      process.nextTick(() => {
+        if (!res.headersSent) {
+          next(err)
```

Comment:
Personally, would recommend keeping them separate rather than blindly passing the settled value to next as it will break with a simple `return {foo: 'bar'}`

Reply:
Make it so!

Reply:
Done - https://github.com/ministryofjustice/fb-runner-node/pull/51